security-analysis/security-analysis-api/src/main/java/com/powsybl/security/tools/SecurityAnalysisTool.java

```diff
@@ -221,15 +224,16 @@ private static SecurityAnalysisExecution buildExecution(ToolOptions options, Sec
     }
 
     private static SecurityAnalysisResult runSecurityAnalysisWithLog(ComputationManager computationManager,
-                                                             SecurityAnalysisExecution execution,
-                                                             SecurityAnalysisExecutionInput input,
-                                                             Path logPath) {
+                                                                     SecurityAnalysisExecution execution,
+                                                                     SecurityAnalysisExecutionInput input,
+                                                                     Path logPath) {
         try {
-            SecurityAnalysisResultWithLog resultWithLog = execution.executeWithLog(computationManager, input).join();
+
+            SecurityAnalysisResult result = execution.execute(computationManager, input).join();
             // copy log bytes to file
-            resultWithLog.getLogBytes()
+            result.getLogBytes()
```

Comment:
So it seems that the logs will be both written here as a raw file, and in the JSON.